kuma/contributions/jinja2/contributions/contribute.html

```diff
@@ -0,0 +1,228 @@
+{% extends "base.html" %}
+
+
+{% block body_attributes %}id="contribute"{% endblock %}
+
+{% block title %}{{ page_title(_('Contribute')) }}{% endblock %}
+
+{% block site_css %}
+    {{ super() }}
+    {% stylesheet 'jquery-ui' %}
+{% endblock %}
+
+{% block site_js %}
+    {{ super() }}
+    <script src="https://checkout.stripe.com/checkout.js"></script>
+    {% javascript 'jquery-ui' %}
+    {% javascript 'contribute' %}
+{% endblock %}
+
+{% block extrahead %}
+    {# TODO: Implement copy & design for the share functionality #}
+    {% set seo_description = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.' %}
+    {% set social_title = _('Support MDN') %}
+    <meta property="og:description" content="{{ seo_description }}">
+    <meta name="description" content="{{ seo_description }}">
+    <meta name="twitter:description" content="{{ seo_description }}">
+    <meta name="twitter:title" content="{{ social_title }}">
+    <meta property="og:title" content="{{ social_title }}">
+{% endblock %}
+
+{% block document_head %}
+    {% with form=form, is_popover=False %}
+        {% include "contributions/includes/contribution-form.html" %}
+    {% endwith %}
+{% endblock %}
+
+{% block contrib_popup %}{% endblock %}
+
+{% block content %}
+
+<div id="contributions-page" class="conributions-page">
```

Comment:
This seems like the main content of the page so, use `<main role="main">` instead of a `div` to improve semantics and a11y

Reply:
Should be fixed in latest PR